script/vsts/platforms/linux.yml

```diff
@@ -30,12 +30,26 @@ jobs:
       - script: npm install --global npm@6.12.1
         displayName: Update npm
 
-      - task: 1ESLighthouseEng.PipelineArtifactCaching.RestoreCacheV1.RestoreCache@1
-        displayName: Restore node_modules cache
+      - task: Cache@2
+        displayName: Cache node_modules
         inputs:
-          keyfile: 'package.json, script/vsts/platforms/linux.yml, **/package-lock.json, !**/node_modules/**/package-lock.json, !**/.*/**/package-lock.json'
-          targetfolder: '**/node_modules, !**/node_modules/**/node_modules'
-          vstsFeed: 'bae1bc26-220d-43c7-a955-4de039370de2'
```

Comment:
#19441 
#10326 
#20511

Reply:
Please remove the spam comment which references incorrect issues. If I report your account that is much worse @Duv4lC4ruso.

Reply:
I've already reported it as spam. I don't see any particularly legitimate activity on the account. Sorry if it's a real person, but the account activity looks like a bot.